samples/subsys/usb/uvc/README.rst

```diff
@@ -0,0 +1,143 @@
+Any software application to read and process video would then be able
+to access the stream.
+
+Requirements
+************
+
+USB is the only requirement for this sensor, as an emulated image
+sensor and MIPI receiver is used to provide a test pattern from
+software.
+
+The USB descriptor configuration is extracted from these drivers, and
+the user does not need to provide them.
+
+Building and Running
+********************
+
+Build the sample application and flash the resulting binaries.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: nrf52840dongle
+   :goals: build flash
+   :compact:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: mini_stm32h734
+   :goals: build flash
+   :compact:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/uvc
+   :board: rpi_pico
+   :goals: build flash
+   :compact:
+
+Upon reboot, the device is expected to be detected by the operating
+system.
+
+On Linux, OSX and BSDs this is visible using the ``dmesg`` and ``lsusb`` commands.
+
+On Windows, this is visible using the "Windows Device Manager" application.
+
+Playing the Stream
+==================
+
+Some example of client to open the video stream on a Linux
+environment, assuming ``/dev/video2`` is your Zephyr demo freshly
+attached.
+
+Play it using `FFplay <https://ffmpeg.org/ffplay.html>`_
+
+.. code-block:: console
+
+   ffplay /dev/video2
+
+Play it using `GStramer <https://gstreamer.freedesktop.org/>`_:
+
+.. code-block:: console
+
+   gst-launch-1.0 v4l2src device=/dev/video2 ! videoconvert ! autovideosink
+
+Play it using `MPV <https://mpv.io/>`_:
+
+.. code-block:: console
+
+   mpv /dev/video2
+
+Play it using `VLC <https://www.videolan.org/vlc/>`_:
+
+.. code-block:: console
+
+   vlc v4l2:///dev/video2
+
+The video device can also be used by web applications and video
+conferencing systems, as it is recognized by the system as a native
+webcam.
+
+On Windows, accessing the video can be done with the default "Camera" application,
+by switching the camera source.
+
+Android and iPad (but not yet iOS) are also expected to work via
+dedicated applications.
+
+Accessing the Video Controls
+============================
+
+On Linux, the ``v4l2-ctl`` command permits to list the supported controls:
+
+.. code-block:: console
+
+   $ v4l2-ctl --device /dev/video2 --list-ctrls
+
+   Camera Controls
+
+                     auto_exposure 0x009a0901 (menu)   : min=0 max=3 default=1 value=1 (Manual Mode)
+        exposure_dynamic_framerate 0x009a0903 (bool)   : default=0 value=0
+            exposure_time_absolute 0x009a0902 (int)    : min=10 max=2047 step=1 default=384 value=384 flags=inactive
+
+   $ v4l2-ctl --device /dev/video2 --set-ctrl auto_exposure=1
+   $ v4l2-ctl --device /dev/video2 --set-ctrl exposure_time_absolute=1500
+
+On Windows, the `VLC <https://www.videolan.org/vlc/>`_ client and `Pot Player <https://potplayer.tv/>`_
+client permit to further access the video controls.
```

Comment:
I think this is not needed to be documented because it is obvious and OS / tools dependent, not related to Zephyr. Anyone want to use a USB camera in an OS needs to know this in that OS. If needed, we can just say this is a UVC camera device so treat it as a UVC camera in the corresponding OS.

Reply:
This is information that could be useful for reproducing the results of this example: showing some of the tooling that was tested and is expected to work.

For instance: does this sample works with [libuvc](https://libuvc.github.io/libuvc/) on MacOSX? I do not know, I did not test using this, if that does not work, better try with one of the tools that were known to work.

This is also about convenience: grouping a lot of information from 3rd-party tools not likely to change soon, that user can copy-paste one after the other until one of the command work, and report what did not work in an issue.

Although, not documenting any of this is easy to do, I'll just delete these lines. :)

Reply:
I disagree. It's a nice addition and the point is to make it easy to go beyond "just" showing the camera feed,.i.e. also let the user directly play with the various *controls*

Reply:
To be clear my reply was to @ngphibang :) @josuah's reply came at virtually the same time

Reply:
Some important point still: instead of saying "use this tool" first,  the focus needs to be on "this is a regular video device, here are some arbitrary examples". Maybe be "like any webcam" repeated through the different sections?

Reply:
Okay, make sense :-)

Reply:
I changed the example organization to put the accent on "just a webcam". Thanks for the feedback!